roles/ntp/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+chrony_port: 123
+chrony_acquisitionport: 1123
+chrony_allow:
+chrony_service_enabled: 'yes'
+chrony_service_state: 'started'
+chrony_service_name: 'chronyd'
+chrony_conf_file: '/etc/chrony.conf'
+chrony_keyfile: '/etc/chrony.keys'
+chrony_driftfile: '/var/lib/chrony/chrony.drift'
+chrony_log: 'tracking measurements statistics'
+chrony_logdir: '/var/log/chrony'
+chrony_maxupdateskew: 100.0
+chrony_dumpdir: '/var/lib/chrony'
+chrony_initstepslew: false
+chrony_initstepslew_threshold: 30
+chrony_initstepslew_servers: 'time.nist.gov'
```

Comment:
Do we want to use time.nist.gov?  It's inaccessible from the HPE network.

Reply:
Oops. No, we don't.

Reply:
Oh, that file needs work. We don't need much of that.